backend/src/test/kotlin/org/loculus/backend/service/submission/UseNewerProcessingPipelineVersionTaskTest.kt

```diff
@@ -32,19 +39,69 @@ class UseNewerProcessingPipelineVersionTaskTest(
         convenienceClient.extractUnprocessedData(pipelineVersion = 1)
         convenienceClient.submitProcessedData(processedData, pipelineVersion = 1)
         useNewerProcessingPipelineVersionTask.task()
-        assertThat(submissionDatabaseService.getCurrentProcessingPipelineVersion(), `is`(1L))
+        assertThat(submissionDatabaseService.getCurrentProcessingPipelineVersion(Organism(DEFAULT_ORGANISM)), `is`(1L))
 
         convenienceClient.extractUnprocessedData(pipelineVersion = 2)
         convenienceClient.submitProcessedData(processedDataWithError, pipelineVersion = 2)
         useNewerProcessingPipelineVersionTask.task()
-        assertThat(submissionDatabaseService.getCurrentProcessingPipelineVersion(), `is`(1L))
+        assertThat(submissionDatabaseService.getCurrentProcessingPipelineVersion(Organism(DEFAULT_ORGANISM)), `is`(1L))
 
         convenienceClient.extractUnprocessedData(pipelineVersion = 3)
         convenienceClient.submitProcessedData(processedData, pipelineVersion = 3)
         useNewerProcessingPipelineVersionTask.task()
-        assertThat(submissionDatabaseService.getCurrentProcessingPipelineVersion(), `is`(3L))
+        assertThat(submissionDatabaseService.getCurrentProcessingPipelineVersion(Organism(DEFAULT_ORGANISM)), `is`(3L))
 
         submissionControllerClient.extractUnprocessedData(numberOfSequenceEntries = 10, pipelineVersion = 2)
             .andExpect(status().isUnprocessableEntity)
     }
+
+    @Test
+    fun `GIVEN the pipeline v for one organism updates THEN the pipeline v for another organism is not updated`() {
```

Comment:
```suggestion
    fun `GIVEN the pipeline version for one organism updates THEN the pipeline version for another organism is not updated`() {
```

Reply:
That'll be more than 120 characters and the linter complains and I think it's not possible to split the function name over multiple lines 😅

Reply:
Maybe I can also just disble the linting rule for that line! Didn't think about that before.

Reply:
Yep, disabling ktlint is perfectly fine here, also: you can cut out the "the"

Reply:
```suggestion
    fun `GIVEN pipeline v for 1 organism updates THEN pipeline v for other organism is not updated`() {
```

something like that works as well for shortening

Reply:
disabled the linter: https://github.com/loculus-project/loculus/pull/3534/commits/4c6a05c2e39424f8b26fbd43865d7c1425525889